### PR TITLE
[Merged by Bors] - Add docs about redundancy

### LINKED ISSUE #2140: the fallback beacon node was considered to be unworkable, but it works well
## Description

I set --beacon-nodes with two endpoints, for example:
```
http://a.a.a.a:5052,http://b.b.b.b:5052
```
when the `a` shutdown, but `b` works well, the validate-node consider it unusable but `b` works well, I can 
`telnet b.b.b.b:5052`
and get 200 response with :

```
curl http://b.b.b.b:5052/eth/v1/node/version

{"data":{"version":"Lighthouse/v1.0.6-f183af2/x86_64-linux"}}
``` 

```
CRIT Error during attestation routine        slot: 231054, committee_index: 1, error: "All endpoints failed http://a.a.a.a:5052/ => RequestFailed(\"Failed to produce an aggregate attestation: Reqwest(reqwest::Error { kind: Request, url: Url { scheme: \\\"http\\\", host: Some(Ipv4(a.a.a.a)), port: Some(5052), path: \\\"/eth/v1/validator/aggregate_attestation\\\", query: Some(\\\"slot=231054&attestation_data_root=0xc8f1e9bd4b0720ee962a361e84316f608d38c945cd0fbbc3f3f0e035b6cf40ec\\\"), fragment: None }, source: hyper::Error(Connect, ConnectError(\\\"tcp connect error\\\", Os { code: 111, kind: ConnectionRefused, message: \\\"Connection refused\\\" })) })\"), http://b.b.b.b:5052/ => RequestFailed(\"No aggregate available for AttestationData { slot: Slot(231054), index: 1, beacon_block_root: 0x49969fe83e3ba1a1ed91e4ca2391f35f345b7340e620f28598973bd69b938206, source: Checkpoint { epoch: Epoch(7219), root: 0xf14e95c11364c46cb37762143f84daef5cf41c95e508372c7ea891359d0cc8c4 }, target: Checkpoint { epoch: Epoch(7220), root: 0x3b641034a8c3802ce886cfd6332f5b8ad536c7e011042686583e521b30af75f7 } }\")", service: attestation
```

## Version

v1.0.6

### PR DESCRIPTION
**Pull request by paulhauner**
## Issue Addressed

- Resolves #2140

## Proposed Changes

Adds some documentation on the topic of "redundancy".

## Additional Info

NA

### CODE REVIEW

File: book/src/redundancy.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+achieves *redundancy*, allowing the validator client to continue to perform its
+duties as long as *at least one* of the beacon nodes is available.
+
+There are a few interesting properties about the list of `--beacon-nodes`:
+
+- *Ordering matters*: the validator client prefers a beacon node that is
+	earlier in the list.
+- *Synced is preferred*: the validator client prefers a synced beacon node over
+	one that is still syncing.
+- *Failure is sticky*: if a beacon node fails, it will be flagged as offline
```
Comment (realbigsean) [inline book/src/redundancy.md:42]: Does it retry the first beacon node after the current slot ends?

File: book/src/redundancy.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+attestations is only performed for validators which [explicitly request
+subscriptions](subscribe-api).
+
+There are 64 subnets and each validator will result in a subscription to *at
+least* one subnet. So, using the two aforementioned flags will result in
+resource consumption akin to running 64+ validators.
+
+## Redundant Eth1 nodes
+
+Compared to redundancy in beacon nodes (see above), using redundant Eth1 nodes
```
Comment (realbigsean) [inline book/src/redundancy.md:91]: Something that I think is worth pointing out:  if you use the `--eth1-endpoints` flag, there is no default fallback to `http://localhost:8545`, so you have to be explicit with localhost whenever you use this flag.   I don't know if this is the same for `--beacon-nodes`

**Review by realbigsean — CHANGES_REQUESTED** [review]
Could I request an addition to FAQ's about redundancy with a link to these docs? They're great :pray:

File: book/src/redundancy.md (hunk 1 above)
Comment (paulhauner) [reply book/src/redundancy.md:42]: Yep, it does! I re-worded the paragraph to express this :)

File: book/src/redundancy.md (hunk 2 above)
Comment (paulhauner) [reply book/src/redundancy.md:95]: Good call, I added some notes :)

**Comment by paulhauner** [comment]
All comments addressed, thank you!

**Review by realbigsean — APPROVED** [review]
:+1:

**Pull request closed without merging**